src/queries/projects.js

```diff
@@ -13,6 +13,7 @@ query Projects {
     allStrapiProjects {
       nodes {
```

Comment:
is this name different from bio name

Reply:
Yeah it is, I was noticing we were using `bio name` which is not available anymore in Strapi (we moved it to just `name`). The test does look for a student's name so this is covered there.

If needed I am happy to make a staging link to compare w prod